test/higher_order_ops/test_invoke_subgraph.py

```diff
@@ -228,6 +228,23 @@ def fn(x, y):
         self.assertEqual(ref, res)
         res.sum().backward()
 
+    def test_dropout(self):
+        @mark_compile_region
+        def gn(x):
+            return torch.nn.functional.dropout(torch.sin(x), p=0.5)
+
+        @mark_compile_region
+        def hn(x):
+            return torch.sin(x)
+
+        def fn(x):
+            return gn(x) + hn(x)
+
+        x = torch.randn(8, requires_grad=True)
+        # Difficult to check the results here because we random does not match
+        # between eager and Triton.
+        res = torch.compile(fn, backend="inductor", fullgraph=True)(x)  # noqa: F841
```

Comment:
1) why did you pick dropout?
2) dropout is kind of interesting. If we have multiple regions that `gn`, do they exhibit the same random behavior? (that would not be good). That is, if dropout decomposes into `x * torch.randn(shape)`, and the graph bakes in the value of torch.randn(shape), then we've got a problem.

Reply:
Added a comment at the beginning of the test.